apps/site/assets/ts/schedule/components/line-diagram/__tests__/lineDiagramData/northStationBidiTest.json

```diff
@@ -0,0 +1,8106 @@
+              "methods": ["Credit/Debit Card", "Cash"],
+              "daily_rate": "Hourly: 30 Min: $8, 1 hr: $15, 1-2 hrs: $22, 2-3 hrs: $26, 3-12 hrs: $30 | Daily Max: $68 | Events: $48 | Early Bird (in by 9 AM, out by 6 PM): $24 | Nights/Weekends: $10"
+            },
+            "note": "Parking garage is located underneath TD Garden.",
+            "name": "North Station Garage",
+            "manager": {
+              "url": "https://www.propark.com/propark-locator2/north-station-garage/",
+              "phone": "617-222-3042",
+              "name": "ProPark",
+              "contact": "ProPark"
+            },
+            "longitude": -71.06214,
+            "latitude": 42.366083,
+            "capacity": {
+              "type": "Garage",
+              "total": 1275,
+              "overnight": "Unknown",
+              "accessible": 38
+            },
+            "address": null
+          }
+        ],
+        "parent_id": null,
+        "note": null,
+        "name": "North Station",
+        "municipality": "Boston",
+        "longitude": -71.06129,
+        "latitude": 42.365577,
+        "is_child?": false,
+        "id": "place-north",
+        "has_fare_machine?": true,
+        "has_charlie_card_vendor?": true,
+        "fare_facilities": [
+          "fare_media_assistant",
+          "fare_vending_machine",
+          "ticket_window"
+        ],
+        "description": null,
+        "closed_stop_info": null,
+        "child_ids": [
+          "70026",
+          "70027",
+          "70205",
+          "70206",
+          "BNT-0000",
+          "BNT-0000-01",
+          "BNT-0000-02",
+          "BNT-0000-03",
+          "BNT-0000-04",
+          "BNT-0000-05",
+          "BNT-0000-06",
+          "BNT-0000-07",
+          "BNT-0000-08",
+          "BNT-0000-09",
+          "BNT-0000-10",
+          "BNT-0000-B2",
+          "door-north-causewaye",
+          "door-north-causeways",
+          "door-north-crcanal",
+          "door-north-crcauseway",
+          "door-north-crnashua",
+          "door-north-tdgarden",
+          "door-north-valenti"
+        ],
+        "bike_storage": ["bike_storage_rack"],
+        "address": "135 Causeway St, Boston, MA 02114",
+        "accessibility": [
+          "accessible",
+          "escalator_both",
+          "elevator",
+          "fully_elevated_platform",
+          "portable_boarding_lift"
+        ]
+      },
+      "route": {
+        "type": 0,
+        "sort_order": 10035,
+        "name": "Green Line E",
+        "long_name": "Green Line E",
+        "id": "Green-E",
+        "direction_names": { "0": "Westbound", "1": "Eastbound" },
+        "direction_destinations": { "0": "Heath Street", "1": "Union Square" },
+        "description": "rapid_transit",
+        "custom_route?": false,
+        "color": "00843D"
+      },
+      "name": "Test Station B",
+      "is_terminus?": false,
+      "is_beginning?": false,
+      "id": "test station b",
```

Comment:
**nitpick(non-blocking):** can you give these more "realistic" id values, and by that I mean no spaces? Seeing the space-y `key` name in the test is weirding me out...